test/generate_tests.py

```diff
@@ -33,7 +33,7 @@ def test_main(self):
 		n_generate = 21
 		output_folder = 'test_data/test_dataset'
 		sys.argv = ['test','test_data/config_dict.py',output_folder,'--n',
-			str(n_generate),'--tf_record','--save_png_too']
```

Comment:
I understand the logic here, but we still want to include something that runs the tf_record in the analysis tests then. Otherwise we'll lose coverage from never checking those lines.

Reply:
OK, now added back --tf_record to the argv if tensorflow imports